wiki/Contests/Official_support/en.md

```diff
@@ -0,0 +1,66 @@
+
+- **Profile badge prizes for first-place winners.** See profile badge design criteria on the [official tournament support article](/wiki/Tournaments/Official_support#profile-badges).
+- **Progress towards *Elite Mapper* user title.** See [contest points](/wiki/Contests/Contest_points) for more details.
+
+## Eligibility
+
+Contests which abide by the following criteria are eligible for official support for **profile badges**:
+
+- **The contest must meet either of these criteria:**
+  - The contest is organised and run by an experienced team of accomplished and reputable staff, as decided by the osu! team.
+  - The contest has successfully run for one iteration without official support.
+- **The contest must have a clearly defined set of rules.**
+- **The contest must accept submissions for only one game mode.** Most users specialize in one game mode, so it's not reliable to evaluate mapping quality across multiple game modes.
+- **The contest must have at least 10 submissions that adhere to the contest's rules.** Contests with fewer valid submissions are not deemed competitive enough.
+- **The contest must have a clearly defined judging criteria based on mapping merit.** These ensure a contest follows an understandable quality standard in line with other mapping contests.
+- **The contest's results must be determined by valuing scores from all judges equally.** Using standard deviation to determine final scores is preferred but not required.
+- **The contest's results must be publicly displayed with judge scores/comments visible.** The judges may be anonymized publicly, but the osu! team may ask for un-anonymized judge details if necessary.
+- **The contest must have at least 5 judges.** Contests with fewer judges allow each judge too much influence on overall results.
+- **A user cannot participate in more than one submission to the contest.**
+- **Submissions to the contest should be anonymized.** This limits judging bias removing name association between a mapper and their map. If a contest has special circumstances that warrant removing anonymity, this rule may be lifted, but otherwise must be followed.
+- **The contest's organisers must not participate in the contest.** This includes both submitting a map *and* judging submissions. Anyone with access to judging information or un-anonymized submissions is considered a "contest organiser".
+
+In addition to the criteria above, contests which abide by these rules are eligible for official support for **[contest points](/wiki/Contests/Contest_points) to earn the *Elite Mapper* user title:**
+
+- **The contest must exclusively use songs listed in osu!'s [Featured Artist library](https://osu.ppy.sh/beatmaps/artists).**
+- **The contest must be published on [osu!'s front page news](https://osu.ppy.sh/home/news) and the [contest listing](https://osu.ppy.sh/community/contests).** For credibility of the *Elite Mapper* title, a contest must have maximum visibility.
```

Comment:
does this entail only an initial announcement or a set of posts like announcement (and should registration already be opened/determined etc?) and results (+ voting open etc?)?

Reply:
from experience I think it should be initial announcement + results announcement, would be nice to clarify that here if that's still the case

Reply:
i think the initial announcement is what's most important, since it's what increases visibility. results are good to have too ofc, but other posts aren't always part of contests